src/kbmod/masking.py

```diff
@@ -2,7 +2,42 @@
 """
 
 import kbmod.search as kb
+from kbmod.search import RawImage
 
+def mask_trajectory(trj, stack, r, bitmask = 1):
+    """ Will apply a circular mask of radius `r` around
+    the trajectory's predicted position in each image.
+
+    Attributes
+    ----------
+    trj : `Trajectory`
+        The trajectory along which to apply the mask.
+    stack : `ImageStack`
+        The stack of images to apply the mask to.
+    r : `float`
+        The radius of the circular mask to apply.
+
+    Returns
+    -------
+    stack : `ImageStack`
+        The stack after the masks have been applied.
+    """
+
+    dt = stack.get_single_image(1).get_obstime() - stack.get_single_image(0).get_obstime()
+    width = stack.get_single_image(0).get_width()
```

Comment:
You could also use the stack's `get_width()` and `get_height()` functions directly.